### DISCUSSION TITLE
I can't seem to get windows to swallow!

### DISCUSSION THREAD

**Discussion by ghost** [hyprwm/Hyprland#5398] [Q&A] [answered]
What am i doing wrong?

misc {
    force_default_wallpaper = 0
    enable_swallow = true
    swallow_regex = ^(kitty)$
}

command(sxiv, mpv, nautilus - from kitty)

Nothing seem to swallow.

**Comment by vaxerski** [comment, marked as answer]
swallowing depends on the process to keep its pid (not fork) - try with e.g. alacritty, that should work
